index.bs

```diff
@@ -459,23 +459,23 @@ Note: |browsing contexts| is a set because a [=shared worker=] can be associated
 
 To <dfn>obtain a set of event names</dfn> given an |name|:
 
-  1. Let |events| be an empty set.
+1. Let |events| be an empty set.
 
-  1. If |name| contains a U+002E (period):
+1. If |name| contains a U+002E (period):
 
-    1. If |name| is the [=event name=] for an event, append |name| to |events|
-       and return [=success=] with data |events|.
+  1. If |name| is the [=event name=] for an event, append |name| to |events|
+     and return [=success=] with data |events|.
 
-    1. Return an [=error=] with [=error code=] [=Invalid Argument=]
+  1. Return an [=error=] with [=error code=] [=Invalid Argument=]
 
-  1. Otherwise |name| is interpreted as representing all the events in a
-     module. If |name| is not a [=module name=] return an [=error=] with
-     [=error code=] [=Invalid Argument=].
+1. Otherwise |name| is interpreted as representing all the events in a
+   module. If |name| is not a [=module name=] return an [=error=] with
+   [=error code=] [=Invalid Argument=].
 
-  1. Append the [=event name=] for each [=event=] in the module with name |name| to
-     |events|.
+1. Append the [=event name=] for each [=event=] in the module with name |name| to
+   |events|.
 
-  1. Return [=success=] with data |events|.
+1. Return [=success=] with data |events|.
 </div>
```

Comment:
Bikeshed wants different indentation here.

Reply:
Not quite; if you have a non-indented list it turns out that you need a blank line before any markup following the list. Anyway, fixed.